packages/static-wado-webserver/lib/middlewares/cors.mjs

```diff
@@ -0,0 +1,16 @@
+import cors from "cors";
+import dicomWebServerConfig from "../dicomWebServerConfig.mjs";
+import isValidViewerConfig from "../util/isValidViewerConfig.mjs";
+
+const viewerOriginConfig = dicomWebServerConfig.externalViewer || {};
+
+const corsOptions = {
+  origin: `${viewerOriginConfig.protocol}://${viewerOriginConfig.host}:${viewerOriginConfig.port}`,
```

Comment:
This is ok for now, but the calculations as to what cors systems to allow should come from a list.